File: backend/src/main/java/cohappy/backend/model/dto/RegisterDTO.java
package cohappy.backend.model.dto;

import cohappy.backend.model.Currency;
import lombok.Data;

import java.util.List;

@Data
public class RegisterDTO {
    private List<byte[]> images;
    private String email;
    private String phoneNumber;
    private String password;
    private Currency currency;
}

File: frontend/app/src/main/java/cohappy/frontend/client/dto/request/RegisterDTO.kt
package cohappy.frontend.client.dto.request

import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class RegisterDTO(
    val name: String,
    val surname: String,
    val birthDate: String,
    var images: List<ByteArray> = emptyList(),
    val email: String,
    val phoneNumber: String,
    val password: String
)
